### ISSUE TITLE
camel forge - Add endpoint vs edit endpoint error in IDEA

### ISSUE THREAD

**Issue by davsclaus** [fabric8io/fabric8#5374]
To add a netty4-http endpoint fails in IDEA. But if you add the endpoint manually and edit it afterwards it works. So its a little bug/difference between the add/edit commands. And potentially maybe that UISelectOne that must have its current value using one from the select list otherwise its an index out of bounds exception

**Comment by davsclaus** [comment]
its the consumer options on camel-netty4 that causes this. Sadly the only error you get is

    10:45:34 AM IndexOutOfBoundsException: Index: 2, Size: 2

So it will take a bit time to track down. The producer options work fine.

**Comment by davsclaus** [comment]
Yay found the stacktrace in the idea.log file
```
2015-12-08 10:45:34,770 [ 148569]  ERROR - llij.ide.plugins.PluginManager - Index: 2, Size: 2 
java.lang.IndexOutOfBoundsException: Index: 2, Size: 2
	at java.util.ArrayList.rangeCheck(ArrayList.java:653)
	at java.util.ArrayList.get(ArrayList.java:429)
	at org.jboss.forge.addon.ui.impl.controller.WizardCommandControllerImpl.getCurrentEntry(WizardCommandControllerImpl.java:491)
	at org.jboss.forge.addon.ui.impl.controller.WizardCommandControllerImpl.getCurrentController(WizardCommandControllerImpl.java:502)
	at org.jboss.forge.addon.ui.impl.controller.WizardCommandControllerImpl.getInputs(WizardCommandControllerImpl.java:244)
	at org.jboss.forge.addon.ui.impl.controller.NoUIWizardControllerDecorator.getInputs(NoUIWizardControllerDecorator.java:189)
```

**Comment by davsclaus** [comment]
Whoa this takes a longer timer to track down as its a bit trial and error

**Comment by davsclaus** [comment]
yay its fixed in forge 3.0 alpha 3 release.
